Algorithm.Python/MeanVarianceOptimizationAlgorithm.py

```diff
@@ -24,8 +24,10 @@
 from QuantConnect.Algorithm.Framework.Execution import *
 from QuantConnect.Algorithm.Framework.Risk import *
 from QuantConnect.Algorithm.Framework.Selection import *
-from Alphas.HistoricalReturnsAlphaModel import *
-from Portfolio.MeanVarianceOptimizationPortfolioConstructionModel import *
+#from Alphas.HistoricalReturnsAlphaModel import *
+#from Portfolio.MeanVarianceOptimizationPortfolioConstructionModel import *
+from QuantConnect.Algorithm.Framework.Alphas import *
+from QuantConnect.Algorithm.Framework.Portfolio import *
```

Comment:
Since you have commented out the imports from python models, this example will use the C# models. When possible, we should use python models in python algorithms.